gulpfile.js

```diff
@@ -1349,7 +1369,6 @@ gulp.task('dist', 'Build production binaries', ['update-packages'], dist, {
     minimal_set: '  Only compile files needed to load article.amp.html',
   }
 });
-gulp.task('extensions', 'Build AMP Extensions', buildExtensions);
 gulp.task('watch', 'Watches for changes in files, re-builds when detected',
     ['update-packages'], watch, {
       options: {
```

Comment:
Add documentation for `--extensions` and `--noextensions` here, and for `gulp` and `gulp build`.

Reply:
done